tensorflow/core/common_runtime/bfc_allocator.cc

```diff
@@ -253,7 +255,8 @@ void* BFCAllocator::AllocateRawInternalWithRetry(
 
 void* BFCAllocator::AllocateRaw(size_t unused_alignment, size_t num_bytes,
                                 const AllocationAttributes& allocation_attr) {
-  VLOG(1) << "AllocateRaw " << Name() << "  " << num_bytes;
+  VLOG(3) << "AllocateRaw " << Name() << "  " << num_bytes;
+  void* result = nullptr;
```

Comment:
I'd prefer keeping the previous structure; with this new flow it is easier to forget to set `result` in some branch.

How about using IIFE:

```
void* result = [&] {
  // Old code that does `return AllocateRawInternalWithRetry` etc.
}();
VLOG(3) << ...
return result;
```

Reply:
done.